_pages/en/product-fw-standard.md

```diff
@@ -0,0 +1,404 @@
+- Performs various HR activities for the Product Team (e.g. staffing, coaching, mentoring)
+- Secures funding through business case(s) and through good pitch in government
+- Ensures the Product Team is equipped for success (e.g. procuring tools, training)
+- Understands the different phases of product delivery
+
+The above taken from [Product Development and Management Association](https://www.pdma.org/page/npdp-bok-second-edition)'s Body of Knowledge second edition, from [280group](https://280group.com/what-is-product-management/roles/product-manager/), [GOV.uk's guidance](https://www.gov.uk/guidance/product-manager).
+
+### Product Owner Role
+
+The Product Owner is a tactical role.
+Will typically look at a time horizon within 3-6 months.
+The Product Owner interfaces with, and engages the business stakeholders.
+
+The Product Owner is the final authority for decisions regarding priority, business value, and functionality for the work done by the Product Development team.
+
+The Product Owner performs the following activities:
+
+- Acts as a fully integrated team member
+- Uses clear goals to communicate the internal project (release scope) vision at the beginning of every release so that the team understands and is equipped to realize it
+- Makes business decisions and clearly communicates through requirements elaboration techniques (e.g. epics and user stories) so they are understood
+- Works closely with IT Managers in taking the technical aspect of each release scope
+- Ensures compliance with IM/IT-related Treasury Board Directives such as Cyber Security software patching
+- Communicates status outside the team
+
+The above from [Info-Tech Research Group](https://www.infotech.com/research/it-product-owner).
+
+### IT Security Champion Role
+
+The IT Security Champion is the go-to-person for IT Security knowledge about the product.
+
+The IT Security Champion performs the following activities:
+
+- Understands the security tools
+- Implements security tools in the pipeline
+- Analyzes security test results
+- Mitigates identified risks in software-based products
+- Promotes and coaches the team members on security best practices
+
+The above from [SLF's January 2021 presentation](https://014gc.sharepoint.com/:p:/r/sites/DGIIT-Gouvernance-IITB-Governance/SLFFCS/DevSecOps SLF-EN.pptx?d=wc4a2579808df446eab5b3b5b3ceedb5f&csf=1&web=1&e=VVS26e) (modified last point)
+
+### Change Coordinator Role
+
+The Change Coordinator ensures that all changes are managed in a controlled manager, including standard changes and emergency maintenance.
+
+The Change Coordinator performs the following activities:
+
+- Performs impact assessments
+- Seeks the necessary authorisations prior to the change (e.g. IITB Release Process)
+- Ensures all stakeholders are aware of the upcoming product release
+- Documents the change
+
+The above from [Info-Tech Research Group](https://www.infotech.com/research/change-manager).

```

Comment:
Ditto above comments. The more I'm seeing this I'm starting to wonder is it the activities only that is from that resource or the entire role definition.